client/src/containers/Topic/TopicList/TopicList.jsx

```diff
@@ -275,6 +275,22 @@ class TopicList extends Root {
               colName: 'Size',
               type: 'text'
             },
+            {
+              id: 'lastWrite',
+              accessor: 'name',
+              colName: 'Last Record Write',
+              type: 'text',
+              cell: (obj, col) => {
+                return this.getApi(uriTopicLastRecord(selectedCluster, obj[col.accessor]))
```

Comment:
Don't do apiCall during rendering of the view, you need to use React State, like for Consumer Group : 

https://github.com/tchiotludo/akhq/blob/9dd39982eb79f4acf2cbdebcbd90a1a8bb2871f9/client/src/containers/Topic/TopicList/TopicList.jsx#L142-L165 

We need to gather all api call in state (to have a full vision of all api call) and it will display a loading mask (done with default undefined value)

Reply:
ack